README.md

```diff
@@ -153,6 +154,11 @@ This variable is optional.  It allows you to exclude a file (e.g. index.html) fr
 This variable is optional.  It allows you to automatically set the title.
 
 
+### `S3_REGION` variable
```

Comment:
I think you'd want to clarify how this interacts with bucket url and bucket website url e.g. if you have this do you not need them?

If so, you might want to merge this item up into the main config and explain that it is an alternative.

Either way a bit more explanation would be great.